src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/ProfessionalUserServiceImpl.java

```diff
@@ -106,6 +110,44 @@ public ProfessionalUser findProfessionalUserByEmailAddress(String email) {
         return professionalUserRepository.findByEmailAddress(RefDataUtil.removeAllSpaces(email));
     }
 
+    @Override
+    public ResponseEntity<Object> findRefreshUsers(String since, Pageable pageable) {
+        DateTimeFormatter formatter = DateTimeFormatter.ISO_DATE_TIME;

```

Comment:
might be better to centralise this as a constant somewhere, so we can use it here + on the validator too? That way we have a bit more control over it if needed

Reply:
addressed in commit [8ea5e96](https://github.com/hmcts/rd-professional-api/pull/1529/commits/8ea5e969fad63b555ae4f3f8928ccfa957f73b68) - made as constant